gum/backend-darwin/gumobjcapiresolver.c

```diff
@@ -7,8 +7,11 @@
  * Licence: wxWindows Library Licence, Version 3.1
  */
 
+#include "guminterceptor.h"
+
 #include "gumobjcapiresolver.h"
 #include "gumobjcapiresolver-priv.h"
+#include "gumobjcdisposeclasspairmonitor.h"
 
 #include "gumprocess.h"
```

Comment:
```suggestion
#include "gumobjcapiresolver.h"

#include "guminterceptor.h"
#include "gumobjcapiresolver-priv.h"
#include "gumobjcdisposeclasspairmonitor.h"
#include "gumprocess.h"
```

(These were already in the wrong order but we should fix it now.)